Main.java

```diff
@@ -0,0 +1,16 @@
+class Main {
+  public static void main(String[] args) {
+
+    int n = 10, firstTerm = 0, secondTerm = 1;
+    System.out.println("Fibonacci Series till " + n + " terms:");
+
+    for (int i = 1; i <= n; ++i) {
+      System.out.print(firstTerm + ", ");
+
+      // compute the next term
+      int nextTerm = firstTerm + secondTerm;
+      firstTerm = secondTerm;
+      secondTerm = nextTerm;
+    }
+  }
+}
```

Comment:
**Here is the explanation of code:**

```- The code starts by declaring a class called Main.
- It then has an instance method main with no arguments, which is the entry point of the program.
- The first line in main declares an int variable n that will be used to keep track of how many terms are being computed.
- Next, it declares two variables: firstTerm and secondTerm that will be used to store the starting values for Fibonacci numbers.
- Then it prints out "Fibonacci Series till" followed by n plus "terms:" followed by 10 lines of code that compute each term in turn until they reach 10 terms (the last line).

- The next part starts with a for loop that iterates through all integers from 1 to n-1 while printing out each term as well as its predecessor on separate lines.
- After this is done, there's another for loop inside of it where i is initialized at 1 and goes up until i equals n-1 before exiting the loop again; this time, however, instead of just printing out one number like before, it computes the next value using two new variables: nextTerm and firstTerm respectively.
- First Term gets set equal to second Term and Second Term gets set equal to Next Term so now we have three different values stored in,- The code is a part of the Main class.

- The Main class has a public static void main(String args) method.

- This method takes in 10 ints, firstTerm, secondTerm and n terms as arguments.
- The System.out object prints out Fibonacci Series till n terms: "Fibonacci Series till" + n + " terms: " + terms: "."
- Then for each i from 1 to n , it prints out the first term followed by the second term, then computes the next term and prints it out before moving on to the next value in line with i .```